.dockerignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
Since you are copying the whole directory, it would be good to make this .dockerignore more comprehensive.

E.g. `.vagrant`, `Dockerfile`, `Vagrantfile`, `.git`

Reply:
Updated, thank you